puppet/zulip/manifests/profile/smokescreen.pp

```diff
@@ -6,16 +6,28 @@
   include zulip::golang
 
   $version = 'dc403015f563eadc556a61870c6ad327688abe88'
+  $dir = "/srv/smokescreen-src-${version}/"
   zulip::sha256_tarball_to { 'smokescreen':
     url     => "https://github.com/stripe/smokescreen/archive/${version}.tar.gz",
     sha256  => 'ad4b181d14adcd9425045152b903a343dbbcfcad3c1e7625d2c65d1d50e1959d',
     install => {
-      "smokescreen-${version}/" => "/srv/smokescreen-src-${version}/",
+      "smokescreen-${version}/" => $dir,
     },
   }
+  file { $dir:
+    ensure  => directory,
+    require => Zulip::Sha256_tarball_to['smokescreen'],
+  }
+  tidy { '/srv/smokescreen-*':
+    path    => '/srv/',
+    recurse => 1,
+    rmdirs  => true,
+    matches => 'smokescreen-*',
```

Comment:
Same comment that we may want to rename to `/srv/zulip-smokescreen` before doing this.